maps-compose/src/main/java/com/google/maps/android/compose/MapApplier.kt

```diff
@@ -15,14 +15,9 @@
 package com.google.maps.android.compose
 
 import androidx.compose.runtime.AbstractApplier
-import androidx.compose.ui.platform.ComposeView
 import com.google.android.gms.maps.GoogleMap
 import com.google.android.gms.maps.MapView
-import com.google.android.gms.maps.model.Circle
-import com.google.android.gms.maps.model.GroundOverlay
-import com.google.android.gms.maps.model.Marker
-import com.google.android.gms.maps.model.Polygon
-import com.google.android.gms.maps.model.Polyline
+import com.google.android.gms.maps.model.*
```

Comment:
Revert to single name imports. This is a reminder for me to add spotless or an equivalent code formatter to prevent these type of changes.